src/pages/stacks-blockchain/configurations.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Configurations
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Installation
+
+See this page for installation instructions.
+[@page-reference | inline]
+| /stacks-blockchain/running-testnet-node
+
+## Usage
+
+`stacks-node sub-command [--subcommand-option <value>]`
+
+## Command-line Subcommands and Options
```

Comment:
This reads like a reference page - could we maybe move this to the reference section instead?

Reply:
Totally! I didn't think of that. Updated.